Setting-up-Rhetos-for-HTTPS.md

```diff
@@ -0,0 +1,54 @@
+# Setting up Rhetos for HTTPS
+
+In order for Rhetos to work on HTTPS you need to add a new endpoint in web.config file.
+
+Insert following code:
+```C
+<security mode="Transport">
+  <transport clientCredentialType="None" />
+</security>
+```
+
+Into:
+```C
+<basicHttpBinding><binding> </binding></basicHttpBinding>
+```
+
+And into:
+```C
+<webHttpBinding><binding> </binding></webHttpBinding>
+```
+
+Example:
+```C
+<services>
+      <service name="Rhetos.RhetosService">
+        <clear />
+        <endpoint address="" binding="basicHttpBinding" bindingConfiguration="rhetosBasicHttpBinding" name="basic" contract="Rhetos.IServerApplication" listenUriMode="Explicit">
+          <identity>
+            <dns value="localhost" />
+            <certificateReference storeName="My" storeLocation="LocalMachine" x509FindType="FindBySubjectDistinguishedName" />
+          </identity>
+        </endpoint>
+    '''<endpoint address="mex" binding="mexHttpsBinding" contract="Rhetos.IServerApplication" />'''

```

Comment:
Please remove all `'''` from the code snippet. Is seems to be an obsolete formatting that somehow got left in the old version of the text.

Reply:
Removed in new version